module1/homework/grayscaleImages/compression.cpp

```diff
@@ -41,7 +41,7 @@ void printMap(const PGMBitMap& img) {
     std::stringstream ss;
     for (const auto& row : img) {
         for (const Pixel pixel : row) {
-            ss << std::setw(4) << std::setfill(' ') << std::to_string(pixel);
+            ss << (char)(isprint(pixel) ? pixel : ' ');
```

Comment:
no need to cast on `char`. BTW this is old C-style cast. Normally we use `static_cast<char>(pixel)`